plugins/flytekit-k8s-pod/flytekitplugins/pod/task.py

```diff
@@ -19,6 +19,14 @@ def _sanitize_resource_name(resource: _task_models.Resources.ResourceEntry) -> s
 
 
 class Pod(object):
+    """
+    Pod is a platform-wide configuration that uses pod templates. By default, every task is launched as a container in a pod.
+    This plugin helps expose a fully modifiable Kubernetes pod spec to customize the task execution runtime.
+    To use pod tasks:
+    1. Define a pod spec;
+    2. Specify the primary container name.
```

Comment:
Since this list is so small so how about we collapse it into a single line, like
```suggestion
    To use pod tasks:  (1) define a pod spec, and (2) specify the primary container name.
```

Reply:
Sure @eapolinario